netbox/virtualization/choices.py

```diff
@@ -11,17 +11,20 @@ class VirtualMachineStatusChoices(ChoiceSet):
     STATUS_ACTIVE = 'active'
     STATUS_OFFLINE = 'offline'
     STATUS_STAGED = 'staged'
+    STATUS_DECOMMISSIONING = 'decommissioning'
 
     CHOICES = (
         (STATUS_ACTIVE, 'Active'),
         (STATUS_OFFLINE, 'Offline'),
         (STATUS_STAGED, 'Staged'),
+        (STATUS_DECOMMISSIONING, 'Decommissioning'),
     )
 
     LEGACY_MAP = {
         STATUS_OFFLINE: 0,
         STATUS_ACTIVE: 1,
         STATUS_STAGED: 3,
+        STATUS_DECOMMISSIONING: 4,
```

Comment:
This is not needed. `LEGACY_MAP` exists only for backward-compatibility with pre-2.7 values; there is no need to populate new values.

Reply:
Thx, removed the line.